tests/Sinch.Tests/Conversation/SendMessageTests.cs

```diff
@@ -500,5 +503,23 @@ public async Task SendAllParams()
 
             response.Should().NotBeNull();
         }
+
+        [Theory]
+        [InlineData("300s", 300)]
+        [InlineData("  300s ", 300)]
+        [InlineData(null, null)]
+        [InlineData("", null)]
+        [InlineData("   ", null)]
+        public void DeserializeTll(string actual, int? expected)
+        {
+            // NOTE: api doesn't return TTL for deserialization anywhere
```

Comment:
In this case, does it make sense to implement a function for it? Can the `Read` function just return null?

Reply:
I would be sad for a customer to build the request as a JSON object to then deserialize it and use the SDK just for sending the request. But let's keep it, the implementation looks fine now 🙌